tools/README.md

```diff
@@ -0,0 +1,16 @@
+# Setup
```

Comment:
consider adding a shell script that runs these commands in a temp directory to make sure they remain valid?

Reply:
you mean as part of the build/release process?

Reply:
yea, that's what i was thinking. there are lots of projects that link to other repos, but those links have gone dead.